docs/reference_guide.md

```diff
@@ -2821,6 +2829,39 @@ Attaching 1 probe...
 /dev/pts/1 -> /dev/pts/1
 ```
 
+## 26. `uptr()`: Annotate userspace pointer
+
+Syntax:
+- `uptr(void *p)`
+
+Annotate `p` as a pointer belonging to userspace address space.
+
+This instructs bpftrace to generate calls to `bpf_probe_read_user*()` when
```

Comment:
I wonder if we should write about internal things (`bpf_probe_read_user`). Maybe just writing bpftrace infers pointer address space but sometimes fails so in this case use it is enough?

Reply:
Yeah, that makes sense